BackofficeBundle/Resources/public/template/Menu/menuView._tpl.html

```diff
@@ -14,35 +14,35 @@
     <% } %>
     <% if (user.access_section.contribution) { %>
         <li>
-            <a href="#menu-contribution" role="tab" data-toggle="tab" id="navigation-contribution">
```

Comment:
Why remove this? Useless here, i'm ok. But open if an integrator want to highlight just this menu enter.